client/plots/matrix.js

```diff
@@ -1117,7 +1117,20 @@ export class Matrix {
 				}
 
 				const key = anno.key
-				const values = anno.filteredValues || anno.values || [anno.value]
+
+				let values
```

Comment:
@congyu-lu why doesn't lines 607-610 or 613 in `classifyValues()` work? In other words, I think we should apply `this.stackSiblingCellsByClass` to the filteredValues also, and not do it within `getSerieses()`

```js
                   else {
			// dt=1 are SNVindels, dt=4 CNV, dt=3 Gene Expression
			// will render only one matching value per dt
			for (const dt of [4, 1, 3]) {
				const v = filteredValues.find(v => v.dt === dt)
				if (v) renderedValues.push(v)
			}
		}
		// group stacked cell values to avoid striped pattern *** !!! apply to filteredValues? !!!!
		if (tw.term.type == 'geneVariant') renderedValues.sort(this.stackSiblingCellsByClass)
```

Reply:
Lines 607-610 only work for renderedValues. I will move the code sort filteredValues based on dt to classifyValues and also apply this.stackSiblingCellsByClass to filteredValues